aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -53,8 +54,8 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
 
     // TODO(Alin): These SHA512 gas costs could be unified with the costs in our future SHA512 module
     // (assuming same implementation complexity, which might not be the case
-    [.ristretto255.sha512_per_byte, "ristretto255.scalar_sha512_per_byte", 1],
-    [.ristretto255.sha512_per_hash, "ristretto255.scalar_sha512_per_hash", 1],
+    [.ristretto255.sha2_512_per_byte, "ristretto255.scalar_sha512_per_byte", 1],
+    [.ristretto255.sha2_512_per_hash, "ristretto255.scalar_sha512_per_hash", 1],
```

Comment:
I suppose gas parameter renames would also be breaking changes, no? @davidiw, @wrwg, @vgao1996?

Reply:
Yep this has to be done in multiple steps for long-lived testnet. Can we pause on these changes for now until we have a release process defined here? ETA: next week

Reply:
If you are changing just the rust name of the gas parameter, but not the on-chain name of it, then it's not a breaking change and you can make it at any time

Reply:
Yep....... :|

Reply:
I think in this case, the best way to proceed is to add a new gas parameter instead of renaming it, leaving the old one unused...